nipype/interfaces/base.py

```diff
@@ -764,6 +764,8 @@ def __init__(self, **inputs):
             raise Exception('No input_spec in class: %s' %
                             self.__class__.__name__)
         self.inputs = self.input_spec(**inputs)
+        self.estimated_memory = 1
```

Comment:
Can we rename this to `estimated_memory_gb` or `estimated_memory_mb` to be more explicit about units?